### ISSUE TITLE
loss cannot go down

### ISSUE THREAD

**Issue by Shawn-CT** [LukasStruppek/Plug-and-Play-Attacks#7]
Hello, I executed this command: python attack.py -c=configs/attacking/default_attacking.yaml, and load Stylegan2 weight pretrained on FFHQ1024, to attack Resnet50 train on Facescrub，but I found that under Adam's optimization, loss could not go down at all. Is this related to the model's lower acc on the test set？
![8a5a9ca85c1a57970172d522f89208b](https://github.com/LukasStruppek/Plug-and-Play-Attacks/assets/44296250/5a3e6fec-ef64-4aa8-878e-a451a295007d)

**Comment by LukasStruppek** [comment]
Hey,

this should be fine, since the mean confidence is increasing and approaching 1.0. The applied Poincaré loss avoids vanishing gradients compared to the standard cross-entropy loss, so it won't saturate.

Best,
Lukas
